code/parse/sexp.cpp

```diff
@@ -12403,6 +12410,50 @@ void sexp_set_player_orders(int n)
 	}
 }
 
+void sexp_set_order_allowed_target(int n)
+{
+	bool allow_order;
+	std::set<size_t> orders;
+
+	auto ship_entry = eval_ship(n);
+	if (!ship_entry || !ship_entry->shipp) {
```

Comment:
Style: there should be a new line before and after if statements to make them distinct from surrounding code.